src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,84 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
+1.4.0.dev0 (5/12/2017)
+----------------------
+
```

Comment:
It looks like these notes contain entries that duplicate the `1.3.0rc0` notes... I think this should probably have been generated by running:
```
 build-support/bin/release-changelog-helper.sh release_1.3.0rc0
```
...to include only changes since the `release_1.3.0rc0` tag.